doc/DHCPv6_relay/DHCPv6-relay-agent-High-Level-Design.md

```diff
@@ -166,33 +169,138 @@ RELAY-REPLY
-			    type bool;
-		    }
+module sonic-dhcpv6-relay {
+
+	namespace "http://github.com/Azure/sonic-dhcpv6-relay";
+
+	prefix sdhcpv6relay;
+
+	yang-version 1.1;
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	organization "SONiC";
+
+	contact "SONiC";
+
+	description "DHCPv6 Relay yang Module for SONiC OS";
+
+	revision 2021-10-30 {
+		description "First Revision";
+	}
+
+	container sonic-dhcpv6-relay {
+
+		container DHCP_RELAY {
+
+			description "DHCP_RELAY part of config_db.json";
+
+			list DHCP_RELAY_LIST {
+
+				key "name";
+        
+				leaf name {
+					type string;
+				}
+
+        leaf-list dhcpv6_servers {
+                description "Configure the dhcpv6 servers";
+                type inet:ipv6-address;
         }
+
+        leaf dhcpv6_option|rfc6939_support {
+                description "Set rfc6939 for the relay";
+        		    type bool;
+		    }
+
+        leaf dhcpv6_option|interface_id {
+                description "Set interface-id for the relay";
+        		    type bool;
+		    }
     }
+  }
 }
 </pre>
 
 # Option 79 for client link-layer address
 
-Option 79 should be enabled by default and can be disabled through command line.
+Option 79 should be enabled by default and can be disabled through editing config_db.json and reapplying configuration or config database directly by the following commands:
+
+Usage:
+```
+redis-cli -n 4 hset DHCP_RELAY|<vlan> dhcpv6_option|rfc6939_support <true/false>
+config vlan dhcp_relay add <vlan_id> <dhcp_relay_destination_ip>
+```
+Restart dhcp_relay after updating ipv6 helper information.
+
+Or edit directly in config_db.json and apply config_reload:
+```
+/etc/sonic/config_db.json:
+{
+        "DHCP_RELAY" :{
+                "Vlan100" : {
+                        "dhcpv6_servers" : ["fc02:2000::2"]
+			"dhcpv6_option|rfc6939_support" : "true"
+                }
+        }
+}
+
+sudo config reload
+```
+
+# Option 18 for Interface-ID
+
+Option 18 is disabled by default and can be enabled through editing config_db.json and reapplying configuration or config database directly by the following commands:
+
+Usage:
+```
+config vlan dhcp_relay add <vlan_id> <dhcp_relay_destination_ip>
+redis-cli -n 4 hset DHCP_RELAY|<vlan> dhcpv6_option|interface_id <true/false>
+```
+Or edit directly in config_db.json and apply config_reload:
+/etc/sonic/config_db.json:
+{
+        "DHCP_RELAY" :{
+                "Vlan100" : {
+                        "dhcpv6_servers" : ["fc02:2000::2"],
+			"dhcpv6_option|interface_id" : "true"
+                }
+        }
+}
+
+sudo config reload
+
+Example:
+```
+admin@sonic:~$ redis-cli -n 4 hset DHCP_RELAY|Vlan1000 dhcpv6_option|interface_id true
+```
+
+# Config Update
+
+We have shifted from VLAN table to DHCP_RELAY table to store ipv6 helper information.
+Only DHCP_RELAY table needs to be updated for the versions containing PR https://github.com/sonic-net/sonic-buildimage/pull/10654, or version 20201231.75 and 20181130.98 and above
```

Comment:
Can you also include this PR https://github.com/sonic-net/sonic-buildimage/pull/13006 and those versions might not work completely as suggested in the  description of the PR